docs/conf.py

```diff
@@ -176,13 +176,21 @@
 html_sidebars = {"**": ["sidebar-nav-bs.html", "sidebar-ethical-ads.html"]}
 
 # For edit button
-html_context = {
-    "github_user": "pybamm-team",
-    "github_repo": "pybamm",
-    "github_version": "develop",
-    "doc_path": "docs/",
-}
+html_context.update(
+    {
+        "github_user": "pybamm-team",
+        "github_repo": "pybamm",
+        "github_version": "develop",
+        "doc_path": "docs/",
+    }
+)
+
+# Set canonical URL from the Read the Docs Domain
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
 
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
```

Comment:
```suggestion
if os.getenv("READTHEDOCS") == "True":
```
This should be ok too, right?